frontend/app/(footer)/layout.tsx

```
@@ -1,14 +1,32 @@
+  title: "SPORTSHUB | Find your next social sport session!",
   description: "Sportshub helps you find and book local sports events.",
 };
 
 export default function RootLayout({ children }: { children: React.ReactNode }) {
   return (
     <>
       {children}
+
+      <script
+        type="application/ld+json"
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)